### PR TITLE
new repository for group M

### PR DESCRIPTION
**Pull request by HAL2328**
We had to create a new repository to integrate a GitHub Organization

### CODE REVIEW

File: repositories.py [hunk 1]
```diff
@@ -87,6 +87,7 @@
         "group m",
         "Group M",
         ["https://github.com/davrepo/devops2024"],
+        ["https://github.com/DevOps2024-Organization/devops2024/tree/main"],
```
Comment (Herover) [inline repositories.py:90]: Either combine the list of repositories or replace the one you are no longer using (if it doesn't contain information not available in the new one)

**Review by Herover — CHANGES_REQUESTED** [review]
Just needs a small change :)

File: repositories.py (hunk 1 above)
Comment (HAL2328) [reply repositories.py]: Thanks! Should be fixed now

**Comment by HAL2328** [comment]
I think this is fixed now. I've added the new repository to the list, I think both should be linked, even though we'll only be updating the new one from now on

**Pull request merged**